app/services/daily_report.rb

```diff
@@ -21,8 +23,7 @@ def posts(sort='')
         range.begin,
         range.end,
       ]))
-      .joins("LEFT JOIN replies AS replies_today ON replies_today.post_id = posts.id")
-      .where("replies_today.created_at IS NULL OR (replies_today.created_at between ? AND ?)", range.begin, range.end)
+      .joins("LEFT JOIN replies AS replies_today ON replies_today.post_id = posts.id AND replies_today.created_at between '#{range.begin}' AND '#{range.end}'")
```

Comment:
Can joins not do the substitution thing? I realize this is probably safe in context but

Reply:
Nope, because Rails hates us and wants us to suffer.